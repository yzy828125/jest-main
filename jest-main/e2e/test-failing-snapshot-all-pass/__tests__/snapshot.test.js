/**
 * Copyright (c) Meta Platforms, Inc. and affiliates.
 *
 * This source code is licensed under the MIT license found in the
 * LICENSE file in the root directory of this source tree.
 */

test.failing('snapshots not updated', () => {
  expect('1').toMatchSnapshot();
  expect('1').toMatchSnapshot();
});
