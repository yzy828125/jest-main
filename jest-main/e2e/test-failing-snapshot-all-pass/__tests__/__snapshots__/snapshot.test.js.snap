// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`snapshots not updated 1`] = `"1"`;

exports[`snapshots not updated 2`] = `"1"`;
